content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -113,34 +94,32 @@ stages:
         md5: ede2872bedbfe10342bb1c416e2f049f
 ```
 
-**Remember: DVC doesn’t replace Git.** Having codified data
-<abbr>artifacts</abbr> and [pipelines](/doc/start/data-pipelines), regular `git`
-commands are used to create versions (commits, tags, branches, etc.):
+> See [Data Pipelines](/doc/start/data-pipelines) to learn about stages.
```

Comment:
Not sure we want to include the dvc.lock example and link to data pipelines, may be too much/ confusing. Should we only have the data.dvc example? It would definitely help make the doc shorter, but it looses some completeness.

Reply:
Plus maybe we want the pipelines teaser.

Reply:
I decided to remove this after all. Resolving for now